models/models.go

```diff
@@ -45,6 +45,7 @@ type Filter struct {
 	LastUpdated     time.Time           `bson:"last_updated"               json:"-"`
 	ETag            string              `bson:"e_tag"                      json:"-"`
 
+	ID         string      `bson:"-"                    json:"id,omitempty"`
```

Comment:
Should this be `bson:"id"?

Reply:
We don't want to persist this id to the Data store. The filter_id is already getting persisted.
We only want to expose it to keep it in parity with other APIs.